src/components/Card.js

```diff
@@ -1,35 +1,61 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleImageClick) {
+  constructor(
+    { name, link },
+    cardSelector,
+    handleImageClick,
+    handleDeleteClick,
+    removeLikeCard,
+    likeCard,
+    _id,
+    isLiked
+  ) {
     this._name = name;
     this._link = link;
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._handleDeleteClick = handleDeleteClick;
+    this._removeLikeCard = removeLikeCard;
+    this._likeCard = likeCard;
+    this.id = _id;
+    this._isLiked = isLiked;
     this._cardEl = null; // Initialize _cardEl to null
   }
 
+  getId() {
+    return this.id;
+  }
+
   _setEventListeners() {
     this._cardImageEl.addEventListener("click", () => {
-      this._handleImageClick({name: this._name, link: this._link});
+      this._handleImageClick({ name: this._name, link: this._link });
     });
 
     this._cardEl
       .querySelector(".card__like-button")
```

Comment:
You've found `this._likeButton` in line 74. Please, use it here

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.